### PR TITLE
Added default endpoint info. Added note about backwards compabitibility

### PR DESCRIPTION
**Pull request by divyaac**

### CODE REVIEW

File: website/content/api-docs/system/inspect/index.mdx [hunk 1]
```diff
@@ -2,13 +2,18 @@
...
 
 # `/sys/internal/inspect`
 
 The `/sys/internal/inspect` family of endpoints is intended to inspect a specific internal subsystem for debugging purposes.
-It can be accessed with a root token or sudo privileges.
+This endpoint is off by default. See the
+[Vault configuration documentation](/docs/configuration) to
+enable. Once the endpoint is turned on, it can be accessed with a root token or sudo privileges.
+
+~> **NOTE**: These endpoints are only available in Vault version 1.13+. Backwards compatibility is not guaranteed. These endpoints are subject to change or may disappear without notice.
```
Comment (mladlow) [inline website/content/api-docs/system/inspect/index.mdx:16]: Would you mind adding this note content to the router page as well? Anything we can do to make these harder to miss is a win.

**Review by mladlow — APPROVED** [review]
Thanks for the updates, looks good!

If you want, you can prefix the branch names for docs PRs like `docs/VAULT-11513` instead of `VAULT-11513` and you'll get to skip a lot of the CI checks.

File: website/content/api-docs/system/inspect/index.mdx [hunk 2]
```diff
@@ -2,13 +2,18 @@
 layout: api
 page_title: /sys/internal/inspect - HTTP API
 description: >-
-  This endpoint is intended to inspect a specific internal subsystem for debugging purposes. It can be accessed with a root token or sudo privileges
+  This endpoint is intended to inspect a specific internal subsystem for debugging purposes, and turned off by default.
```
Comment (raskchanky) [inline website/content/api-docs/system/inspect/index.mdx:5]:
```suggestion
  This endpoint is intended to inspect a specific internal subsystem for debugging purposes, and is turned off by default.
```

File: website/content/api-docs/system/inspect/index.mdx [hunk 3]
```diff
@@ -2,13 +2,18 @@
...
+  Once turned on, it can be accessed with a root token or sudo privileges
```
Comment (raskchanky) [inline website/content/api-docs/system/inspect/index.mdx:6]:
```suggestion
  Once turned on, it can be accessed with a root token or sudo privileges.
```

**Review by raskchanky — APPROVED** [review]
Two tiny nits, looks great!

**Pull request merged**